textfile.sh

```diff
@@ -0,0 +1,4 @@
+echo "Please enter a word"
+read word
+echo "YOu have entereted" $word "eord"
```

Comment:
@Lakshmikanthwodeyar "eord" is wrong, can you please correct it to word.

Reply:
@appars reviews are changed. thanks for the help.